python/packages/ai/teams/ai/planner/response_parser.py

```diff
@@ -67,6 +67,20 @@ def parse_json(text: str) -> dict:
     return obj
 
 
+def parse_adaptive_card(text: str) -> Optional[dict]:
```

Comment:
we might want to move the parsers out of the planner since this parser isn't a part of the planner logic.